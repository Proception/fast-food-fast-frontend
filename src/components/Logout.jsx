import React, { Component } from 'react';
import RestaurantControls from './RestaurantControls.jsx';
import RestaurantMenu from './RestaurantMenu.jsx';

export default class Restaurant extends Component {

    componentDidMount(){
        localStorage.removeItem('authToken');
        window.location.replace('/');
    }
  render() {
    return (
      <div className="main-container">
        <h2> Logging out</h2>
    </div>
    )
  }
}
