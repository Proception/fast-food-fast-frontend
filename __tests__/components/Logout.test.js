import React from 'react';
import { shallow } from 'enzyme';
import Logout from '../../src/components/Logout.jsx';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Logout />);
});
describe('Logout Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
