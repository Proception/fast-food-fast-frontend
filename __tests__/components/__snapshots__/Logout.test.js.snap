// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logout Component should match snapshot 1`] = `
<div
  className="main-container"
>
  <h2>
     Logging out
  </h2>
</div>
`;
